Sources/MapboxMaps/Ornaments/DistanceFormatter.swift

```diff
@@ -11,20 +11,27 @@ internal class DistanceFormatter: MeasurementFormatter {
     ///
     /// - parameter distance: The distance, measured in meters.
     /// - returns: A localized formatted distance string including units.
-    internal func string(fromDistance distance: CLLocationDistance) -> String {
+    internal func string(fromDistance distance: CLLocationDistance, useMetricSystem: Bool? = nil) -> String {
 
         numberFormatter.roundingIncrement = 0.25
 
         var measurement = Measurement(value: distance, unit: UnitLength.meters)
 
-        if !locale.usesMetricSystem {
+        let shouldUseMetricSystem: Bool
+        if let useMetricSystem = useMetricSystem {
+            shouldUseMetricSystem = useMetricSystem
+        } else {
+            shouldUseMetricSystem = locale.usesMetricSystem
+        }
+
+        if !shouldUseMetricSystem {
```

Comment:
Nit: My personal preference but can we revert the logic here? It's a bit easier to read the `if statement` than `if NOT statement` for me.
What do you think?